graphql/schema.graphql

```diff
@@ -12,6 +12,7 @@ type Query {
     testName: String = ""
     status: String = ""
   ): [TestResult!]
+  testFiles(taskId: String!): [GroupedFiles!]
```

Comment:
Do we maybe want [GroupedFiles!]! to avoid handling a null array?

Reply:
wouldn't this be `taskFiles`?

Reply:
good one